coalib/parsing/DefaultArgParser.py

```diff
@@ -72,6 +72,12 @@ def default_arg_parser(formatter_class=None):
                             help='Files that will be analyzed will be '
                                  'restricted to those in the globs listed '
                                  'in this argument as well the files setting')
+    arg_parser.add_argument('--caching',
```

Comment:
it would probably good to have several categories of arguments to make it easier for users to view them, does argparse support that? I'd rather add a `--cached` and `--flush-cache` argument seperately